docs/en/getting_started/example_datasets/metrica.md

```diff
@@ -0,0 +1,51 @@
+# import data
+cat hits_v1.tsv | clickhouse-client --query "INSERT INTO datasets.hits_v1 FORMAT TSV"
```

Comment:
Let's add `--max_insert_block_size=100000` to limit memory consumption.
(it's required for servers with 4 GB RAM or less)